### PR TITLE
[Bugfix-12504] Have caseSensitive & operators refer to each other

### PR DESCRIPTION
**Pull request by livecodealice**
Added mutual referencing between the caseSensitive entry and related operator entries

### CODE REVIEW

File: docs/notes/Bugfix-12504.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Added mutual referencing between the caseSensitive entry and related operator entries
```
Comment (livecodepanos) [inline docs/notes/Bugfix-12504.md]: This file needs to be renamed to `docs/notes/bugfix-12504.md` :)

**Comment by livecodepanos** [comment]
@livecode-vulcan review ok 702423b

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok 702423b8f96685a61cc6ed6e1903703060ef28f2

**Comment by livecode-vulcan** [comment]
😎 test success 702423b8f96685a61cc6ed6e1903703060ef28f2

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/707)
* try-community-armv7-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-ndk16r15/builds/116)
* try-community-arm64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-ndk16r15/builds/116)
* try-community-x86-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-ndk16r15/builds/116)
* try-community-x86_64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-ndk16r15/builds/116)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2233)
* try-community-universal-ios-iphoneos13.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos13.2/builds/8)
* try-community-universal-ios-iphonesimulator13.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator13.2/builds/8)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2262)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2249)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2248)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2282)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2222)

**Pull request merged**